File: twidere/src/main/java/org/mariotaku/twidere/fragment/support/InvalidTabFragment.java
package org.mariotaku.twidere.fragment.support;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import org.mariotaku.twidere.R;

public class InvalidTabFragment extends BaseSupportFragment {

	@Override
	public View onCreateView(final LayoutInflater inflater, final ViewGroup container, final Bundle savedInstanceState) {
		return inflater.inflate(R.layout.invalid_tab, container, false);
	}

}

File: twidere/src/main/kotlin/org/mariotaku/twidere/fragment/InvalidTabFragment.kt
package org.mariotaku.twidere.fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import org.mariotaku.twidere.R

class InvalidTabFragment : BaseSupportFragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_invalid_tab, container, false)
    }

}
